plans/tests.fmf

```diff
@@ -10,6 +10,9 @@ prepare:
     script:
       - sudo dnf install -y 'dnf-command(copr)'
       - sudo dnf copr enable -y @storage/udisks-daily
+      # TF prioritizes Fedora tag repo over all others, in particular our daily COPR
+      - for f in $(grep -l -r 'testing-farm-tag-repository' /etc/yum.repos.d); do sed -i '/priority/d' "$f" ;done
+      - sudo dnf -y update
```

Comment:
Are you worried/did you observe that the test setup installed packages from the t-f repository which are *newer* than your COPR? We have often seen this in Cockpit, e.g. when there is a mass rebuild in rawhide and all the revisions are suddenly -2 instead of -1. Then you need `distro-sync` here, not `update`.

Reply:
> when there is a mass rebuild in rawhide and all the revisions are suddenly -2 instead of -1

We have a workaround for this -- our packit daily builds have release `-99` to make sure we are always newer even with all the rebuilds.